packages/server/README.md

```diff
@@ -140,7 +140,83 @@ OpenFeature.setProvider(new MyProvider());
 Once the provider has been registered, the status can be tracked using [events](#eventing).
 
 In some situations, it may be beneficial to register multiple providers in the same application.
-This is possible using [domains](#domains), which is covered in more details below.
+This is possible using [domains](#domains), which is covered in more detail below.
+
+#### Multi-Provider
```

Comment:
This is a reminder to myself to update the readme template and docs parser. It would be nice to have this in the support matrix.